XcodeServerSDK/XcodeServer.swift

```diff
@@ -415,6 +447,32 @@ public extension XcodeServer {
         }
     }
     
+    /**
+    XCS API call for retrievieng all available integrations on server.
+    
+    - parameter integrations:   Optional array of integrations.
+    - parameter error:          Optional error.
+    */
+    public func getIntegrations(completion: (integrations: [Integration]?, error: NSError?) -> ()) {
```

Comment:
Btw, you can also use *filters* for integrations, see `/Applications/Xcode-beta.app/Contents/Developer/usr/share/xcs/xcsd/routes/routes_integration.js`. Not that you have to add it now, just FYI, probably something to add at some point in the future :wink: